File: Java/program13.java
import java.util.*;

class DisplayClass
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        for(iCnt = 1; iCnt <= 4; iCnt++)
        {
              System.out.println(iNo);  
        }
     


    }
}

class program13
{
    public static void main(String A[])
    {
        int iValue = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter value to be display :");
        iValue = sobj.nextInt();

        DisplayClass dobj = new DisplayClass();
        dobj.Display(iValue);

        
    }
}

File: C/program13.c
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Display Hello on output screen 
// Autor       :  Ankita Anil Patil
// Date        :  05/05/2025
///////////////////////////////////////////////////////////////////////////////


#include<stdio.h>

//Iteration

void Display(iNo)
{
    int iCnt=0;

    for(iCnt = 1; iCnt <= 5; iCnt++ )
    {
        printf("%d\n",iNo);
    }
}

int main()
{
    int iValue=0;

    printf("Enter Value to be display :\n" );
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}